bricksrc/quantities.py

```diff
@@ -1,109 +1,565 @@
+                QUDT.hasDimensionVector: QUDTDV["A0E0L0I0M1H0T0D0"],
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("Grains"),
+                SKOS.broader: QUDTQK.Mass,
+            }
+        },
+    },
+    "Phasor": {
+        "subconcepts": {
+            "PhasorAngle": {
+                QUDT.applicableUnit: [
+                    UNIT.ARCMIN,
+                    UNIT.ARCSEC,
+                    UNIT.DEG,
+                    UNIT.GON,
+                    UNIT.GRAD,
+                    UNIT.MIL,
+                    UNIT.RAD,
+                    UNIT.MicroRAD,
+                    UNIT.MilliRAD,
+                    UNIT.MilliARCSEC,
+                    UNIT.REV,
+                ],
+                QUDT.hasDimensionVector: QUDTDV["A0E0L0I0M0H0T0D1"],
+                SKOS.definition: Literal("Angle component of a phasor"),
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("PhasorAngle"),
+                SKOS.broader: QUDTQK.PlaneAngle,
+            },
+            "PhasorMagnitude": {
+                QUDT.applicableUnit: [
+                    UNIT.ARCMIN,
+                    UNIT.ARCSEC,
+                    UNIT.DEG,
+                    UNIT.GON,
+                    UNIT.GRAD,
+                    UNIT.MIL,
+                    UNIT.RAD,
+                    UNIT.MicroRAD,
+                    UNIT.MilliRAD,
+                    UNIT.MilliARCSEC,
+                    UNIT.REV,
+                ],
+                QUDT.hasDimensionVector: QUDTDV["A0E0L0I0M0H0T0D1"],
+                SKOS.definition: Literal("Magnitude component of a phasor"),
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("PhasorMagnitude"),
+            },
+        },
     },
-    "Grains": {},
     "Power": {
-        "subclasses": {
+        OWL.sameAs: QUDTQK["Power"],
+        "subconcepts": {
             "Electric_Power": {
-                "subclasses": {
-                    "Apparent_Power": {},
-                    "Active_Power": {OWL.equivalentClass: BRICK["Real_Power"]},
-                    "Real_Power": {},
-                    "Reactive_Power": {},
-                    "Complex_Power": {},
+                OWL.sameAs: QUDTQK["ElectricPower"],
+                "subconcepts": {
+                    "Apparent_Power": {OWL.sameAs: QUDTQK["ApparentPower"]},
+                    "Active_Power": {
+                        OWL.sameAs: [QUDTQK["ActivePower"], BRICK["Real_Power"]],
+                    },
+                    "Real_Power": {
+                        OWL.sameAs: [QUDTQK["ActivePower"], BRICK["Active_Power"]],
+                    },
+                    "Reactive_Power": {OWL.sameAs: QUDTQK["ReactivePower"]},
+                    "Complex_Power": {OWL.sameAs: QUDTQK["ComplexPower"]},
                 },
             },
             "Peak_Power": {
```

Comment:
It might be a separate discussion, but I'd suggest removing vague definitions like this and related Points. (Still, Peak_Power may mean Power_Rating, which has a specific definition)

Reply:
`Peak_Power` and `Power_Rating` are different things

Reply:
I was referring to this: https://en.wikipedia.org/wiki/Peak_power.

Reply:
Ok, so it sounds like it should stay? We've provided a definition that I think is clear and matches how the term is used in the built environment domain